### ISSUE TITLE
Can the minimum threshold for knowledge base annotation be lowered?

### ISSUE THREAD

**Issue by kevintsai1202** [langgenius/dify#11748]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Some embedding models yield relatively low scores, and setting the minimum annotation threshold at 0.8 often fails to achieve matches. Could the minimum threshold be lowered to 0.7?

### 2. Additional context or comments

![圖片](https://github.com/user-attachments/assets/6363e555-ea9e-4197-bced-a07cee0c12a0)

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by crazywoola** [comment]
https://github.com/langgenius/dify/blob/f96fdc2970fe408ca3d671f38b1db25e4959858e/web/app/components/base/features/new-feature-panel/annotation-reply/score-slider/index.tsx#L21-L43

**Comment by kevintsai1202** [comment]
> https://github.com/langgenius/dify/blob/f96fdc2970fe408ca3d671f38b1db25e4959858e/web/app/components/base/features/new-feature-panel/annotation-reply/score-slider/index.tsx#L21-L43
> 
> Actually you can, but I would like to know the number you would like.

Currently, testing with 0.7 achieves better matching.

**Comment by kevintsai1202** [comment]
Is there a chance to implement this enhancement?

**Comment by kevintsai1202** [comment]
> Hi, [@kevintsai1202](https://github.com/kevintsai1202). I'm [Dosu](https://dosu.dev), and I'm helping the Dify team manage their backlog. I'm marking this issue as stale.
> 
> **Issue Summary**
> 
>     * You suggested lowering the minimum threshold for knowledge base annotation from 0.8 to 0.7.
> 
>     * This change is due to low scores from some embedding models affecting effective matching.
> 
>     * User crazywoola mentioned that the threshold can be adjusted and asked for the desired number.
> 
>     * You tested with a 0.7 threshold and found it yielded better results.
> 
>     * You inquired about the possibility of implementing this enhancement.
> 
> 
> **Next Steps**
> 
>     * Is this issue still relevant to the latest version of the Dify repository? If so, please comment to keep the discussion open.
> 
>     * Otherwise, this issue will be automatically closed in 15 days.
> 
> 
> Thank you for your understanding and contribution!

Yes, the latest version requires this adjustment. If it's possible to set the minimum value in the .env environment, that would also work.
